_includes/pages/sponsors/top_sponsors.html

```diff
@@ -0,0 +1,10 @@
+<ul class="cards-list">
+  {% for sp in site.data.sponsors limit:3 %}
```

Comment:
Yeah, that's just a stand in for any more elaborate mechanism.
Since these are so special, I think we could perhaps just make a explicit data list for enterprise sponsors.

Adding more is no problem. It'll just break into a new row.

Reply:
Ah there's actually a `level` field in `sponsors.csv`. So I suppose we could use that to filter all sponsors with an appropriate level (> 1000?).
And Manas would be an extra item?

Reply:
> Ah there's actually a level field in sponsors.csv. So I suppose we could use that to filter all sponsors with an appropriate level (> 1000?).

sounds good. we can tweak it later if we are in the lucky situation.

> And Manas would be an extra item?
Yes, we need to add Manas. I'm tracking this in #423

Reply:
I already added an extra item for manas in 0c7bbedbdfb16254246f3021a09e4d901cb30782
https://deploy-preview-428--crystal-website.netlify.app/sponsors/
![grafik](https://user-images.githubusercontent.com/466378/225417780-9eee69d5-d21e-49b8-a575-018f64e42867.png)

Details are up for improvement of course =)